frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx

```diff
@@ -17,14 +17,15 @@ import { NodesSelectionList } from './nodes-selection-list';
 import { State, Action } from './state';
 import {
   MAX_DISK_SIZE,
+  diskSizeUnitOptions,
   diskModeDropdownItems,
   diskTypeDropdownItems,
-  diskSizeUnitOptions,
+  allNodesSelectorTxt,
 } from '../../constants';
 import './create-local-volume-set.scss';
 
 export const LocalVolumeSetInner: React.FC<LocalVolumeSetInnerProps> = (props) => {
```

Comment:
```suggestion
export const LocalVolumeSetInner: React.FC<LocalVolumeSetInnerProps> = ({ dispatch, state, diskTypeOptions, diskModeOptions, allNodesHelpTxt }) => {
```

Reply:
either of the approach is fine. Since the props are more, hence I prefer to destructure inside the component.